openapi.yaml

```
@@ -0,0 +1,712 @@
+          description: Internal server error _(service cannot be explained)_
+      security:
+        - subscription_key: []
+  /services/{serviceId}/release:
+    post: 
+      tags:
+        - service-release
+      summary: Publish service on IO
+      description: |
+        Publish service by `serviceId` on __IO Platform__. <br><br>_Note: This API operation needs the __Manage API key__ owned by the service
+        user, otherwise `403 forbidden` will be returned to the caller._
+      operationId: releaseService
+      parameters:
+        - name: serviceId
+          in: path
+          description: ID of the service
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation _(service published)_
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not found
+        '412':
+          description: Precondition failed _(service status is incompatible with release request)_
+        '429':
+          description: Too many requests
+        '500':
+          description: Internal server error _(service cannot be published)_
+      security:
+        - subscription_key: []
+    get: 
+      tags:
+        - service-release
+      summary: Retrieve last published version of service
+      description: |
+        Publish service by `serviceId` on __IO Platform__. <br><br>_Note: This API operation needs the __Manage API key__ owned by the service
+        user, otherwise `403 forbidden` will be returned to the caller._
+      operationId: getPublishedService
+      parameters:
+        - name: serviceId
+          in: path
+          description: ID of the service
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation _(service published)_
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Service'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not found
+        '429':
+          description: Too many requests
+      security:
+        - subscription_key: []
+    delete: 
+      tags:
+        - service-release
+      summary: Unpublish service from IO
+      description: |
+        Unpublish service by `serviceId` from __IO Platform__. <br><br>_Note: This API operation needs the __Manage API key__ owned by the
+        service user, otherwise `403 forbidden` will be returned to the caller._
+      operationId: unpublishService
+      parameters:
+        - name: serviceId
+          in: path
+          description: ID of the service
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation _(service unpublished)_
```

I propose the following response code:
```suggestion
        '204':
          description: Service unpublished
```